vite.config.js

```diff
@@ -1,12 +1,12 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 import path from 'path';
-import getEnvModule from './env'
 import dns from 'dns'
+import dotenv from 'dotenv'
 
-dns.setDefaultResultOrder('verbatim')
+dotenv.config()
```

Comment:
The env variables are already loaded from memory with `process.env`. However, I left this clause in case users wanted to run the frontend without Docker, so that a `.env` file can be used in this folder

Reply:
Please revert this change to the behavior in the `main` branch. If the behavior in the `main` branch does not break any required functionality, we will want to keep the code as close as possible to `main` so that we can continuously pull in upstream changes easily. Lets limit ourselves to only adding required functionality.